lib/atmosphere.rb

```diff
@@ -97,6 +97,10 @@ def self.ability_class
     @@ability_class.constantize
   end
 
+  mattr_accessor :nics
+  mattr_reader :nics
```

Comment:
This is not needed since [`mattr_accessor`](http://apidock.com/rails/Module/mattr_accessor) creates getter and setter